src/forms/upload/upload.html

```
@@ -71,8 +71,15 @@ <h4 class="m-b-2" ng-if="!$ctrl.isTooLarge && $ctrl.errorMessage">{{$ctrl.errorM
         <span class="icon icon-alert icon-xxl text-danger m-b-1"></span>
       </div>
       <div ng-if="$ctrl.hasTranscluded" ng-transclude></div>
-      <p ng-if="$ctrl.cancelText" class="m-t-2 m-b-0">
-        <a href="" ng-click="$ctrl.clear()">{{$ctrl.cancelText}}</a>
+      <p class="m-t-2 m-b-0">
+        <a href="" ng-click="$ctrl.clear()">
+          <span 
+            ng-if="!$ctrl.cancelText" 
+            class="glyphicon glyphicon-remove-circle" 
```

Is there anything in the `icon` set we can use?  Ideally we wouldn't use glyphicons, once we switch the caret over to something else, it would be good to remove them overall and save the transport cost.

---

Looks we have something in here, but it's not a dependency.
https://transferwise.github.io/icons/

And here (this is a dependency, but only for the search icon in select):
https://transferwise.github.io/iconfont/

I don't really want to make importing an additional icon font a requirement, just to get one glyph.  I wonder if we can inline the SVG from the former.  I would also potentially do it for the search icon, and remove the icon dependency altogether.

(The select can also use icons within options, but these are non-essential and will fail silently if the icon font is not available)

---

Or if you can use the SVG from the former repo, without bundling the whole thing?

---

Sure I can extract the **svg** from transferwise/icons

Another thought is to use placeholder text like "Remove this file" but I didn't see a precedence for using translations in the dynamic-flow... 

I'll move towards extracting the **svg** though, but open to thoughts on using a placeholder text.

---

@StevePole I updated the code to use `<svg>` from the transferwise/icons repo

I also updated the screenshot for a missing `cancelText` since it's a tad different icon.